config/options.go

```diff
@@ -539,6 +543,14 @@ func (o *Options) Validate() error {
 		o.AuthenticateURL = u
 	}
 
+	if o.SignOutRedirectURLString != "" {
```

Comment:
Can we do this validation check in authenticate only?

Reply:
I think no, because we should also use it in proxy, let see my latest commit.